catalog/installGit.sh

```diff
@@ -12,14 +12,15 @@ source "$CATALOG_HOME/util.sh"
 echo Installing Git package.
 
 createPackage github \
+    -p endpoint "openwhisk.ng.bluemix.net"
     -a description "Package which contains actions and feeds to interact with Github"
 
 waitForAll
 
 install "$CATALOG_HOME/github/webhook.js" \
     github/webhook \
     -a feed true \
-    -a description 'Creates a webhook on github to be notified on selected changes' \
+    -a description 'Creates a webhook on github to be notified on selected changes or deletes the webhook' \
```

Comment:
i don't think we should document the "Deletes the web hook" part, as this is an internal part of open whisk, not something relevant to end users. in other words, when they delete the trigger, the webhook will be deleted, this is just open whisk functioning as it should..

the intent of this description field is to help end users understand the offering.